.github/workflows/pre_release.yaml

```diff
@@ -24,21 +28,20 @@ jobs:
         with:
           release_type: prerelease
 
-  lint_check:
-    name: Lint check
-    uses: apify/workflows/.github/workflows/python_lint_check.yaml@main
-
-  type_check:
-    name: Type check
-    uses: apify/workflows/.github/workflows/python_type_check.yaml@main
-
-  unit_tests:
-    name: Unit tests
-    uses: apify/workflows/.github/workflows/python_unit_tests.yaml@main
+  wait_for_checks:
+    name: Wait for code checks to pass
+    runs-on: ubuntu-latest
+    steps:
+      - uses: lewagon/wait-on-check-action@v1.3.4
```

Comment:
Note to self: This one works well in the `pre_release` case, but for `release`, we need a more robust way of checking the status of the target branch - if there is a [skip ci] commit at the top, it will be automatically considered green, which is not optimal.

Reply:
Do we really? If the [skip ci] commit is a changelog update (it should be in 99% of cases), that only happens after successful code checks, so it should be safe to assume that is green as well.